aruco_tracker:
  ros__parameters:
    aruco_id: 89
    dictionary: 16 # cv::aruco::DICT_ARUCO_ORIGINAL
    marker_size: 1.0
